app/controllers/api/v1/v1_application_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class V1ApplicationController < ApplicationController
```

Comment:
This should be:

```
module Api
  module V1
    class ApplicationController < ::ApplicationController
```

I _think_ `::ApplicationController` will do the trick there, worth trying.